api.ts

```diff
@@ -261,6 +262,11 @@ export interface SyncTableDef<
   entityName?: string;
   /** See {@link DynamicOptions.defaultAddDynamicColumns} */
   defaultAddDynamicColumns?: boolean;
+  /**
+   * See {@link SyncTableOptions.propertyAutocomplete}
+   * @hidden
+   */
+  propertyAutocomplete?: MetadataFormula;
```

Comment:
The naming proposed here is "propertyAutocomplete" for this type of autocomplete, vs parameter autocomplete for the kind we already have

If you imagine a future where we let people "autocomplete" pack objects directly into a canvas then maybe "schemaAutocomplete" would be a more accurate/flexible name, but that would feel kind of abstract